README.md

```diff
@@ -0,0 +1,108 @@
+# gmusic-ui.js
+[![Build Status](https://travis-ci.org/gmusic-utils/gmusic-ui.js.svg?branch=master)](https://travis-ci.org/gmusic-utils/gmusic-ui.js)
+[![GitHub release](https://img.shields.io/github/tag/gmusic-utils/gmusic-ui.js.svg)]()
+[![Code Climate](https://img.shields.io/codeclimate/github/gmusic-utils/gmusic-ui.js.svg)]()
+[![GitHub license](https://img.shields.io/github/license/gmusic-utils/gmusic-ui.js.svg)]()
+
+Browser-side JS library for controlling [Google Music][] through UI manipulation.
+
+[Google Music]: https://play.google.com/music/
+
+`gmusic-ui.js` is not created by, affiliated with, or supported by Google Inc.
+
+[Google Music]: https://play.google.com/music/listen
+
+## Getting Started
+It's important to note the `gmusic-ui.js` depends on [`gmusic.js`](https://github.com/gmusic-utils/gmusic.js) in order to operate correctly
+
+### npm
+Install the module with: `npm install gmusic.js gmusic-ui.js`
+
+Once installed, add it to your HTML and access it via `window.GMusic`.
+
+```html
+<script>
+  require('gmusic.js')
+  require('gmusic-ui.js')
+  window.gmusic = new window.GMusic(); // Our Google Music API
+</script>
+```
+
+### Vanilla
+If you are not using a package manager, download the latest script at:
+
+https://raw.githubusercontent.com/gmusic-utils/gmusic-ui.js/master/dist/gmusic-ui.min.js
+
+Then, add it to your HTML and access it via `window.GMusic`.
+
+```html
+<script src="gmusic.min.js"></script>
+<script src="gmusic-ui.min.js"></script>
+<script>
+  window.gmusic = new window.GMusic(window); // Our Google Music API
+</script>
+```
+
+## Documentation
+`gmusic-ui.js` **extends** the `GMusic` constructor, `window.GMusic`
+
+### Data Structure
+
+#### Playlist
+
+```js
+{
+  "id": String,           // Unique ID for this playlist
```

Comment:
There was a comment that seems to have been buried. We should probably move these code blocks to unordered lists for consistency with other JS definitions. Otherwise, we can run into issues like truncated descriptions